packages/terra-consumer-nav/src/Layout.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
```

Comment:
+1, layout is built by the consumer app based on nav components from this project